test/server/overload_manager_impl_test.cc

```diff
@@ -478,18 +525,28 @@ constexpr char kReducedTimeoutsConfig[] = R"YAML(
     - name: envoy.resource_monitors.fake_resource1
   actions:
     - name: envoy.overload_actions.reduce_timeouts
-      typed_config:
-        "@type": type.googleapis.com/envoy.config.overload.v3.ScaleTimersOverloadActionConfig
-        timer_scale_factors:
-          - timer: HTTP_DOWNSTREAM_CONNECTION_IDLE
-            min_timeout: 2s
       triggers:
         - name: "envoy.resource_monitors.fake_resource1"
           scaled:
             scaling_threshold: 0.5
             saturation_threshold: 1.0
+      typed_config:
+        "@type": type.googleapis.com/envoy.config.overload.v3.ScaleTimersOverloadActionConfig
+        timer_scale_factors:
+          - timer: HTTP_DOWNSTREAM_CONNECTION_IDLE
+            min_timeout: 2s
+          - timer: TRANSPORT_SOCKET_CONNECT
+            min_scale: { value: 10 } # percent
   )YAML";
 
+// These are the timer types according to the reduced timeouts config above.
+constexpr std::pair<OverloadTimerType, Event::ScaledTimerMinimum> kReducedTimeoutsMinimums[]{
```

Comment:
nit: Please move this list of cases inside the relevant test case to avoid having a large separation between these definitions and the test body.

Reply:
This list matches exactly what kReducedTimeoutsConfig produces, so I wanted to keep them close together. Please push back if that's not a good enough reason.

Reply:
I don't know which of the two separations is more problematic.  I would consider having both in the test case so everything is in 1 place, or move OverloadManagerImplTest.TimerTypesProduceCorrectMinimums so it is just below this list definition.

Reply:
Moved the test case; that's a good option.